src/main/config/parameter_group_ids.h

```diff
@@ -106,6 +106,7 @@
 #define PG_VTX_CONFIG 515
 #define PG_SONAR_CONFIG 516
 #define PG_ESC_SENSOR_CONFIG 517
+#define PG_I2C_CONFIG 518
 #define PG_BETAFLIGHT_END 517
```

Comment:
`PG_BETAFLIGHT_END` must be dumped when adding items. All this should perhaps be an enum, with a last `PG_BETAFLIGHT_COUNT` item.?